include/z64player.h

```diff
@@ -665,7 +665,7 @@ typedef struct WeaponInfo {
 #define PLAYER_STATE1_14 (1 << 14)
 #define PLAYER_STATE1_15 (1 << 15)
 #define PLAYER_STATE1_16 (1 << 16)
-#define PLAYER_STATE1_17 (1 << 17)
+#define PLAYER_STATE1_PARALLEL (1 << 17)
```

Comment:
since we're finally naming these what do you think about attaching a comment to them? getting a description on mouse hover in the IDE is really nice. especially for this one when a new person will have no clue what is meant by "parallel". once you are familiar with the camera modes and more player systems it makes sense, but even a comment saying "this is tied to `CAM_MODE_TARGET` (which i assume we're renaming back to PARALLEL soon? lol)  and is the camera / control system used for Z-Targeting"  would go a long way.